### PR TITLE
fix: kafka rpc exception not resolved

### PR DESCRIPTION
**Pull request by Inoir**
## PR Type
Fix the one liner

- [ x] Bugfix

Issue Number:
[10379](https://github.com/nestjs/nest/pull/10379)
[10113](https://github.com/nestjs/nest/issues/10113)

## What is the new behavior?
Pull request mentioned remove lines. Not needed to fix this bug.
Other information is in issue

## Does this PR introduce a breaking change?
- [x ] No

### CODE REVIEW

File: packages/microservices/server/server-kafka.ts [hunk 1]
```diff
@@ -220,6 +220,7 @@ export class ServerKafka extends Server implements CustomTransportStrategy {
             reject(err);
```
Comment (kamilmysliwiec) [inline packages/microservices/server/server-kafka.ts:220]: If this condition `err instanceof KafkaRetriableException && !isPromiseResolved` is true then Promise will both reject and resolve (which technically doesn't make much sense)

**Comment by Tony133** [comment]
Is this PR the same as this https://github.com/nestjs/nest/pull/10379 ?

**Comment by Inoir** [comment]
> Is this PR the same as this #10379 ?

yes but without the unneeded line removal. since there is no fix coming. i made new with only the line needed fixed

**Comment by kamilmysliwiec** [comment]
lgtm

**Pull request merged**